static/js/people.js

```diff
@@ -690,6 +690,20 @@ exports.get_realm_users = function () {
     return Array.from(active_user_dict.values());
 };
 
+exports.get_realm_admins = function () {
```

Comment:
I don't know if this function should go into the exports. But, if it does, we should definitely have tests for this.

Reply:
ok